components/epaxos/src/instance/instance.rs

```diff
@@ -21,12 +21,12 @@ pub enum InstanceStatus {
     Executed,
 }
 
-pub struct DepCmds(Vec<InstanceID>);
+pub struct InstIDs(Vec<InstanceID>);
 
 pub struct Instance {
     pub status: InstanceStatus,
-    pub cmds: Vec<command::Command>,
-    pub ballot: replica::BallotNum,
+    pub cmds: Vec<Command>,
+    pub ballot: BallotNum,
     pub seq: u64,
-    pub deps: DepCmds,
+    pub deps: InstIDs,
```

Comment:
```suggestion
    pub deps: Vec<InstanceID>,
```
貌似定义一个包含Vec<InstanceID>的元祖结构体意义不大, 而且用起来还得 InstIDs.0去访问, 比较麻烦

Reply:
改成type alias吧
```
pub type InstIDs = Vec<InstanceID>;
```